ramls/circulation.raml

```diff
@@ -309,8 +309,92 @@ resourceTypes:
           200:
             body:
               application/json:
-                type: !include schema/circulation-rule-matches.json
-                example: !include examples/circulation-rule-matches.json
+                type: !include schema/circulation-rule-loan-policy-matches.json
+                example: !include examples/circulation-rule-loan-policy-matches.json
+          422:
+            description: "Invalid id error."
+            body:
+              application/json:
+                type: error
+                example: !include examples/circulation-rules-id-error.json
+          500:
+            description: "Internal server error"
+            body:
+              text/plain:
+                example: "Internal server error"
+    /request-policy:
```

Comment:
Given new APIs, we should probably be bumping the raml version number here.

Reply:
Yes, this should be a minor increment of the `circulation` interface